website/content/docs/integrations/consul/acl.mdx

```diff
@@ -285,6 +285,12 @@ Similarly to binding rules, namespace rules have a [`Selector`][] expression to
 determine when the rule should be applied and a [`BindNamespace`][] value that
 defines the namespace used.
 
+Auth methods with a namespace rule create Consul tokens in that Consul
+namespace. Binding rules with `-bind-type role` will also target a role (and
+associated policies) in that same Consul namespace. So you should create the
```

Comment:
```suggestion
namespace. Binding rules with `-bind-type role` also target a role and
associated policies in that same Consul namespace. So you should create the
```
present tense

I removed the parentheses because readers don't always read what's inside the parentheses.